### PR TITLE
MED-1538 : Change placeholder language

### PR DESCRIPTION
**Pull request by mariannemedeo**

### CODE REVIEW

File: src/components/Select.js [hunk 1]
```diff
@@ -1,136 +1,142 @@
...
-flex:1;
-list-style: none;
-line-height: 1.5;
-${p => p.active === true && css`background: ${p.theme.cream}`}
-`
+	flex: 1;
+	list-style: none;
+	line-height: 1.5;
+	${p =>
+		p.active === true &&
```
Comment (bpacaud) [inline src/components/Select.js:24]: indent is strange here

File: src/components/Select.js [hunk 2]
```diff
@@ -313,18 +347,16 @@ const Select = ({ children, id, label, required, placeholder, onChange, fallback
...
 
 Select.defaultProps = {
 	readOnly: false,
 	id: null,
 	required: false,
-	placeholder: "Select an option",
-	fallback: "No result found"
-}
+	placeholder: 'Choisir une option',
+	fallback: 'No result found',
```
Comment (bpacaud) [inline src/components/Select.js]: Should you change the fallback message too ?

**Review by bpacaud — COMMENTED** [review]
There are plenty of modifications. It's due to prettier ? The best was before or after ?

File: src/components/Select.js (hunk 1 above)
Comment (mariannemedeo) [reply src/components/Select.js:24]: prettier ...

File: src/components/Select.js (hunk 2 above)
Comment (mariannemedeo) [reply src/components/Select.js]: true

**Review by bpacaud — APPROVED** [review]

**Comment by Chzzs** [comment]
closing in favor of #25

**Pull request closed without merging**